Branch-SDK/src/io/branch/referral/ShareLinkManager.java

```diff
@@ -166,7 +166,7 @@ private void createShareDialog(List<SharingHelper.SHARE_WITH> preferredOptions)
             textView.setTextAppearance(context_, android.R.style.TextAppearance_Medium);
             textView.setTextColor(context_.getResources().getColor(android.R.color.darker_gray));
             textView.setPadding(leftMargin, padding, padding, padding);
-            shareOptionListView.addHeaderView(textView);
+            shareOptionListView.addHeaderView(textView, null, false);
```

Comment:
Instead of changing this constructor can we do 
`textView.setClickable(false);`
Please note that this constructor is used in user defined Title views also where they control the behavior of short title (Line # 160)